RecoTracker/PixelVertexFinding/plugins/alpaka/PixelVertexProducerAlpaka.cc

```diff
@@ -61,6 +64,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
               conf.getParameter<double>("eps"),
               conf.getParameter<double>("errmax"),
               conf.getParameter<double>("chi2max")),
+        maxVertices_(conf.getParameter<int>("maxVertices")),
```

Comment:
Ii think we can lower the default (Run3) here to something like `256` (if we want to keep powers of 2, `128` could happen for some very high PU event) given we have

<img width="1099" alt="image" src="https://github.com/user-attachments/assets/ec25d8a1-cb0b-4d9c-97e6-f8a5ee43e3dc">

[in a recent TTbar PU 2024 RelVal](https://tinyurl.com/2ars67bb). Then we could have:

- `512` for Phase2;
- `32` for HIN;

I've put the changes in https://github.com/AdrianoDee/cmssw/commit/a28e5490b62e03ca97a904005ad9277cc332c995.

Reply:
The `1024` was there to be super safe for Phase2. Maybe too much safe.

Reply:
It doesn't need to be a power of 2 :-)